static/app/views/replays/details.tsx

```diff
@@ -32,8 +38,19 @@ type Props = RouteComponentProps<
 >;
 
 function ReplayDetails({params: {replaySlug}}: Props) {
-  useReplayPageview('replay.details-time-spent');
+  const config = useLegacyStore(ConfigStore);
+  const location = useLocation();
   const organization = useOrganization();
+
+  useReplayPageview('replay.details-time-spent');
+  useRouteAnalyticsEventNames('replay_details.viewed', 'Replay Details: Viewed');
```

Comment:
Will this override the "Page View: xyz" event?

Reply:
The last time I checked, the "Page View" event was still being fired. However, I observed that we often utilize this event to send additional properties for route analysis. I can investigate the method used to trigger the "Page View" event and potentially disable it specifically for the Replay Details page. Would you be interested in that approach?

Reply:
No, i think consistant Page View events is nice to have. 

i think i have to findout more about how all these event hooks and helpers work because i just havn't dug in at all before.